markdown/ch4.markdown

```diff
@@ -1349,9 +1312,7 @@ for n, (signal, label) in enumerate([(v_single, r'$v_0 [Volt]$'),
     axes[n].grid()
 
 axes[2].set_xlim(0, t_ms[-1])
-axes[2].set_xlabel('Time [ms]')
-plt.show()
```

Comment:
Is our policy not to have these?  That'll bamboozle users in IPython (terminal).